cmd/genkeypair/main.go

```diff
@@ -0,0 +1,27 @@
+package main
```

Comment:
Please add a short readme file that makes it clear that:
* This tool is meant for development only
* This tool may have breaking changes every so often